### DISCUSSION TITLE
Does not recognize "client"

### DISCUSSION THREAD

**Discussion by AltRavenAR** [Rapptz/discord.py#9400] [Q&A]
The code does not recognize client. Did discord update to something new?

``
import os
import discord

intents = discord.Intents.default()
intents.message_content = True

client = discord.Client(intents=intents)

@client.event
async def on_ready():
	print('we have logged in as {0.user}'.format(client))

@client.command()
async def add(ctx, left: int, right: int):
    """Adds two numbers together."""
    await ctx.send(left + right)

@client.event
async def on_message(message):
	if message.author==client.user:
		return
	if message.content.startswith('$hello'):
		await message.channel.send('Hello!')
		

client.run(os.environ['TOKEN'])
``

**Comment by lightlybakedman** [comment]
What do you mean "doesn't recognise"?

**Comment by AltRavenAR** [comment]
the console says "AttributeError: 'Client' object has no attribute 'command'"

**Reply by lightlybakedman** [reply]
Yeah, discord.Client doesn't and never did support commands. See discord.ext.commands.Bot if you want to make commands.
Here's an example: Here is the example in the official discord.py repo:
https://github.com/Rapptz/discord.py/blob/master/examples/basic_bot.py
